src/components/nokkelinfo.tsx

```diff
@@ -56,44 +71,43 @@ const Overblikk = () => {
-    const talespraak: StringOrNothing = tolk?.talespraak;
+    const telefon: StringOrNothing = person?.telefon?.find((entry) => entry.prioritet === '1')?.telefonNr;
+    const taletolk: OrNothing<TilrettelagtKommunikasjonData> = tolk;
     const sivilstatus: StringOrNothing = person?.sivilstandliste?.[0]?.sivilstand;
-    const brukersMaal: StringOrNothing = oppfolgingsstatus?.hovedmaalkode;
-    const ytelserAktivitetsFase: StringOrNothing = ytelser?.vedtaksliste?.[0]?.aktivitetsfase; // Henter kun ut 1 ytelse, flere kan være registrert
-    const geografiskEnhet: OrNothing<Enhet> = person?.geografiskEnhet;
+    const hovedmaal: OrNothing<Hovedmal | ArenaHovedmalKode> = oppfolgingsstatus?.hovedmaalkode;
+    const ytelserVedtakstype: StringOrNothing = ytelser?.vedtaksliste
+        ?.map((obj) => obj.vedtakstype)
+        .join(', ');
     const registrertAv: StringOrNothing = registrering?.registrering?.manueltRegistrertAv?.enhet?.navn;
     const datoRegistrert: StringOrNothing = registrering?.registrering?.opprettetDato;
-    const spraak: StringOrNothing = person?.malform;
-
-    const oppfolgingsenhetIDNAVN: StringOrNothing = oppfolgingsenhet?.enhetId + ' ' + oppfolgingsenhet?.navn;
-    const geografiskenhetIDNAVN: StringOrNothing = geografiskEnhet?.enhetsnummer + ' ' + geografiskEnhet?.navn;
+    const serviceGruppe: OrNothing<ArenaServicegruppeKode> = oppfolgingsstatus?.servicegruppe;
+    const MAX_ALDER_BARN = 21;
+    const barn: PersonsBarn[] = person?.barn && person.barn.filter(enkeltBarn => kalkulerAlder(new Date(enkeltBarn.fodselsdato)) < MAX_ALDER_BARN) || [];
 
     return (
         <Panel border className="overblikkPanel">
             <Heading spacing level="2" size="large">
-                Overblikk
+                Nøkkelinfo
             </Heading>
             <BodyLong className="overblikkContainer">
-                <EnkeltInformasjon header="Veileder" value={veilederNavn} />
-                <EnkeltInformasjon header="Telefon" value={telefon} />
-                <EnkeltInformasjon header="Antall barn" value={antallBarn} />
-                <EnkeltInformasjon header="Oppfølgingsenhet" value={oppfolgingsenhetIDNAVN} />
-                <EnkeltInformasjon header="Talespråk" value={talespraak} />
-                <EnkeltInformasjon header="Sivilstatus" value={sivilstatus} />
-                <EnkeltInformasjon header="Brukers mål" value={brukersMaal} />
-                <EnkeltInformasjon header="Ytelse(r)" value={ytelserAktivitetsFase} />
-                <EnkeltInformasjon header="Geografisk enhet" value={geografiskenhetIDNAVN} />
-                <EnkeltInformasjon header="Registrert av" value={registrertAv} />
-                <EnkeltInformasjon header="Dato registrert" value={datoRegistrert} />
-                <EnkeltInformasjon header="Språk" value={spraak} />
+                <EnkeltInformasjon header="Telefon" value={telefon ? telefon : EMDASH} />
```

Comment:
Fordi `EnkeltInformasjon` inneholder en `h3`-tag og `BodyLong` blir til en `p`-tag vil dette resultere i en consol-error som sier noe sånn som at `h3`-tag ikke kan være en chilld av en `p`-tag. Dette bør fikses.

Tips: for å se hvilke HTML-tags som er lovlige childs til en `p`-tag så står dette under "Permitted content"-seksjonen i denne lenken: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p.